ansible/pbTestScripts/testJDK.sh

```diff
@@ -1,14 +1,11 @@
 #!/bin/bash
 
-mv /vagrant/pbTestScripts/workspace/build/src/build/linux-x86_64-normal-server-release/images/jdk8* ~
+mv $HOME/openjdk-build/workspace/build/src/build/linux-x86_64-normal-server-release/images/jdk8* $HOME
 export TEST_JDK_HOME=$(find ~ -maxdepth 1 -type d -name "*jdk8u*"|grep -v ".*jre.*")
-cd $HOME && mkdir -p testLocation
-cd testLocation && git clone https://github.com/adoptopenjdk/openjdk-tests
-cd openjdk-tests
-./get.sh -t $HOME/testLocation/openjdk-tests
-cd TestConfig
-make -f run_configure.mk
+mkdir -p $HOME/testLocation
+[ ! -d $HOME/testLocation/openjdk-tests ] && git clone https://github.com/adoptopenjdk/openjdk-tests $HOME/testLocation/openjdk-tests
+$HOME/testLocation/openjdk-tests/get.sh -t $HOME/testLocation/openjdk-tests
+cd $HOME/testLocation/openjdk-tests/TKG && make -f run_configure.mk
```

Comment:
We probably want to explicitly fail if the `cd` fails, not just skip this one command so
```
cd $HOME/testLocation/openjdk-tests/TKG || exit 1
make -f run_configure.mk
```
Without this it'll still try to run `make compile` and `make _MachineInfo` from the wrong directory if the clone fails for example

Reply:
Good point - Changes have been made :-)